doc/source/skips/1-governance.rst

```diff
@@ -145,16 +146,18 @@ are made according to the following rules:
   sentence (but no change of the scikit-image.org landing page or the “about”
   page), require approval by a core developer *and* no disagreement or requested
   changes by a core developer on the issue or pull request page (lazy
-  consensus). Core developers are expected to give “reasonable time” to others
-  to give their opinion on the pull request if they’re not confident others
-  would agree.
+  consensus). Core developers are expected to wait one or two days for others to give
+  their opinion if they’re not confident that others would agree.
 
-- **Code changes and major documentation changes** require agreement by *two*
-  core developers *and* no disagreement or requested changes by a core developer
-  on the issue or pull-request page (lazy consensus).
+- **Code and major documentation changes, and changes to the API** require agreement by
+  *two* core developers *and* no disagreement or requested changes by a core developer
+  on the issue or pull-request page (lazy consensus). In case of such changes, core
```

Comment:
Not sure how to interpret "in case of such changes:"

* in case of such requests?
* in case the requested changes have been applied?

Reply:
I think @soupault didn't feel that "in this case" was clear enough? I don't mind either version. To me both just reference the described changes above clearly enough.

Reply:
I've never seen pull request written with a dash. If this is not a grammar thing I'd say "pull request" is the more consistent one with the docs. But feel welcome to apply the patch. :)

Reply:
> I've never seen pull request written with a dash. If this is not a grammar thing I'd say "pull request" is the more consistent one with the docs. But feel welcome to apply the patch. :)

My suggestion (above) is to replace "pull-request" with "pull request" precisely. Grammar-wise, it's definitely "a pull request" (noun); in "pull request page" (above) though, the noun is "page" and "pull request" acts as an adjective, that's why it wouldn't be left-field to hyphenate it (look up "compound adjectives"). Still, I suggest that we don't (in this case). :wink:

Reply:
> I think @soupault didn't feel that "in this case" was clear enough? I don't mind either version. To me both just reference the described changes above clearly enough.

Ok, thanks. Then I would say "in the case of disagreement or requested changes" to be clear; I agree with @soupault ("in this case" could mean "agreement by *two* core developers *and* no disagreement or requested changes (...)" since it's a single sentence/chunk right before "in this case").